crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -89,6 +89,14 @@ Examples:
     )]
     rpc_cors_domains: Vec<String>,
 
+    #[arg(
+        long = "rpc.root-version",
+        long_help = "Version of the JSON-RPC API to serve on the / (root) path",
```

Comment:
Maybe all valid values should be listed here?

Reply:
`clap` does that automatically.

`--help` output looks like this:

```
      --rpc.root-version <RPC_ROOT_VERSION>
          Version of the JSON-RPC API to serve on the / (root) path
          
          [env: PATHFINDER_RPC_ROOT_VERSION=]
          [default: v03]
          [possible values: v03, v04]
```